src/view/com/composer/Composer.tsx

```diff
@@ -560,6 +560,7 @@ export const ComposePost = observer(function ComposePost({
                   onChange={setLabels}
                   hasMedia={hasMedia}
                 />
+                <View style={[styles.spacer]} />
```

Comment:
I think it might make more sense here to just change the fragment to a `<View>` and use `style={{gap: 6}}`. Otherwise seems like a good change!

Reply:
Done. Had to change the gap from 6 to 14, since for the existing gap in the `topbarInner` was making the 6px spacer act like a 14px spacer